lib/aws-rails-provisioner/db_cluster.rb

```diff
@@ -223,7 +223,7 @@ def initialize(engine, options = {})
 
       def _default_family
         case @engine
-        when 'AURORA_POSTGRESQL' then 'aurora-postgresql9.6'
+        when 'AURORA_POSTGRESQL' then 'aurora-postgresql11'
```

Comment:
These seem likely to get out of date quickly - potentially out of scope here but I'm guessing we could call rds to get a list

Reply:
Yes. The real question is if we want to take a dependency here. I would think it's okay.. I imagine that these don't move too quickly and don't necessarily have to be up to date.

Reply:
We already require the aws-sdk-rds here.  They shouldn't move too quickly, but its just a place we'd potentially have to keep updating over time - but I think out of scope for this change.

Reply:
Can be done like this:

Aws> rds.describe_db_engine_versions(engine: 'aurora-postgresql').db_engine_versions.map(&:db_parameter_group_family).reverse.uniq